packages/mask/src/plugins/ITO/SNSAdaptor/hooks/useAvailabilityComputed.ts

```diff
@@ -7,9 +7,10 @@ import { useQualification } from './useQualification'
 import { ITO_CONTRACT_BASE_TIMESTAMP } from '../../constants'
 
 export function useAvailabilityComputed(payload: JSON_PayloadInMask | JSON_PayloadFromChain) {
-    const chainId = useChainId()
+    const chain_id = useChainId()
+    const chainId = payload.chain_id ?? chain_id
```

Comment:
The reason the payload uses snake-case is it is an external data format. In project code, we prefer camelCase.